examples/rust/japanese_alley/src/main.rs

```diff
@@ -86,20 +72,40 @@ fn main() -> anyhow::Result<()> {
 
     let image_lens =
         LensBuilder::for_input_column(EntityPathFilter::all(), "foxglove.CompressedImage:message")
-            // TODO: We leave out the `format` column because the `png` contents are not a valid MIME type.
+            // TODO(grtlr): We leave out the `format` column because the `png` contents are not a valid MIME type.
             .add_component_column(
                 EncodedImage::descriptor_blob(),
                 [
                     Op::access_field("data"),
-                    Op::func(list_binary_to_list_uint8),
+                    Op::func(conversions::list_binary_to_list_uint8),
+                ],
+            )
+            .build();
+
+    // TODO(michael): add support for frame_id.
+    let video_lens =
+        LensBuilder::for_input_column(EntityPathFilter::all(), "foxglove.CompressedVideo:message")
+            .add_component_column(
+                VideoStream::descriptor_codec(),
+                [
+                    Op::access_field("format"),
+                    Op::func(conversions::list_string_to_list_codec_uint32),
+                ],
+            )
+            .add_component_column(
+                VideoStream::descriptor_sample(),
+                [
+                    Op::access_field("data"),
+                    Op::func(conversions::list_binary_to_list_uint8),
                 ],
             )
             .build();
 
     let lenses_sink = LensesSink::new(GrpcSink::default())
         .with_lens(image_lens)
         .with_lens(instance_pose_lens)
-        .with_lens(instance_poses_lens);
+        .with_lens(instance_poses_lens)
+        .with_lens(video_lens);
 
     let (rec, _serve_guard) = args.rerun.init("rerun_example_japanese_alley")?;
```

Comment:
I would prefer a better name for this example. It _can_ load the japanese alley MCAP, but it's actually also capable of loading other datasets, now also with the video type that's not in the alley example. What do you think?

Maybe `custom_mcap_lenses`?